freemocap/fmc_runme.py

```diff
@@ -279,6 +287,8 @@ def RunMe(sessionID=None,
 
     # %% Stage 5 - Use Blender to create output data files
     if stage <=5:
+
+
```

Comment:
this is what happens when I leave a comment and click the "start a review" button